### PR TITLE
clinic: revise clinic plan replayer load

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: clinic/clinic-collect-sql-query-plan.md [hunk 1]
```diff
@@ -116,3 +116,7 @@ roles = [
 ```bash
 diag collect <cluster-name> --profile=<profile-filepath> --explain-sql=<statement-filepath>
 ```
+
+### 将结果导入到 TiDB 集群
+
+采集结果中的 `plan_replayer.zip` 可以通过 sql 的方式直接导入到 TiDB 集群中进行信息查看，具体方法可参考[使用 `PLAN REPLAYER` 导入集群信息](/sql-plan-replayer.md#使用-plan-replayer-导出集群信息)
```
Comment (Oreoxmt) [inline clinic/clinic-collect-sql-query-plan.md:122]:
```suggestion
采集结果中的 `plan_replayer.zip` 可以通过 `PLAN REPLAYER LOAD` 语句直接导入到 TiDB 集群中进行信息查看，具体方法可参考[使用 `PLAN REPLAYER` 导入集群信息](/sql-plan-replayer.md#使用-plan-replayer-导入集群信息)
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by hawkingrei — APPROVED** [review]

**Pull request merged**